src/psp_scraper.py

```diff
@@ -0,0 +1,105 @@
+import sys
+import re
+import multiprocessing as mp
+import json
+import requests
+from bs4 import BeautifulSoup
+from bs4.element import Comment
+
+
+def main():
+    with open(sys.argv[1], "r") as input_file:
+        input_data = json.load(input_file)
+
+    url_list = []
+    for project in input_data["projects"]:
+        if project["url"]:
```

Comment:
this will throw a key error if the key does not exist. A better way would be to write `if project.get("url")` as this would catch cases where the URL key does not exist. As it is now, it only applies to cases where the URL key exists but the value is blank.

Reply:
This may be important later as it could save a step in preproccessing data

Reply:
On second thought this is fine, for now we want it to break if we get bad input.